infra/platforms/improve.py

```diff
@@ -48,6 +51,7 @@ def main():
     parser.add_argument('--num-runs', help='number of times to run drivers to obtain an average [100 by default]', type=int)
     parser.add_argument('--py-sample', help='uses a Python based sampling method. Useful for debugging', action='store_const', const=True, default=False)
     parser.add_argument('--key', help='unique identifier under which to place plots and other output', type=str)
+    parser.add_argument('--ablation', help='options: [nc, nl, ncl]', action='store_true')
```

Comment:
I'm assuming `store_true` means the flag is enabled or not enabled?

Reply:
That's what the docs say it does unless i misread